examples/application/data/landDrainageConsent.ts

```diff
@@ -712,6 +712,6 @@ export const landDrainageConsent: Application = {
         payable: [],
       },
     },
-    schema: `https://theopensystemslab.github.io/digital-planning-data-schemas/${version}/schema.json`,
+    schema: `https://theopensystemslab.github.io/digital-planning-data-schemas/${version}/schemas/application.json`,
```

Comment:
When we re-organised the file structure, we missed updating existing example payload URLs - so those are handled here! Sorry it makes for another 13 x 2 files changed though :see_no_evil: